railties/test/application/rake/multi_dbs_test.rb

```diff
@@ -786,7 +786,9 @@ class TwoMigration < ActiveRecord::Migration::Current
         cache_tables_a = rails("runner", "p ActiveRecord::Base.connection.schema_cache.columns('books')").strip
         assert_includes cache_tables_a, "title", "expected cache_tables_a to include a title entry"
 
-        cache_size_b = rails("runner", "p AnimalsBase.connection.schema_cache.size", stderr: true).strip
+        output = rails("runner", "p AnimalsBase.connection.schema_cache.size").split("\n")
+        assert_match(/Ignoring .*\.yml because it has expired/, output[0])
```

Comment:
Is this test really expecting to always see an expired schema cache? Is that an intended/necessary part of the test?

Reply:
I think the answers are "Yes, we should expect to see an expired cache in this case", and "No it's not a necessary part of this test".

So I've dropped this commit in favor of #49859 to discuss how to approach that over there.